tailwind.config.js

```diff
@@ -14,5 +28,5 @@ module.exports = {
       pointerEvents: ['disabled'],
     },
   },
-  plugins: [require('@tailwindcss/forms')],
+  plugins: [require('@tailwindcss/forms'), require('@tailwindcss/custom-forms')],
```

Comment:
Curious why you brought in the `@tailwindcss/custom-forms`, isn't that one superseded by `@tailwindcss/forms`?

Reply:
Not really. tailwindcsd/forms resets styles of some basic input types. tailwindcss/custom-forms adds some custom utility classes. The purposes are different and I just checked that custom-forms does not include the reset that forms has